activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+*   Adapts virtual attributes on `ActiveRecord::Persistence#becomes`.
+
+    When source and target classes have a different set of attributes adapts
+    attributes such that the extra attributes from target are added.
+
+    ```ruby
+    class Person < ApplicationRecord
+    end
+
+    class WebUser < Person
```

Comment:
Hi @intrip, should this be `User` instead, as that is what we pass to `becomes`?
```suggestion
    class User < Person
```

Reply:
Fixed by e45947328efee37f28b964b3bd50dfb603e23d48. :+1:

Reply:
Thanks! 🙂

Reply:
Thanks!